src/test/src/sc/fiji/bdvpg/bdv/SpimDataDisplayDemo.java

```diff
@@ -0,0 +1,34 @@
+package sc.fiji.bdvpg.bdv;
+
+import bdv.util.BdvHandle;
+import sc.fiji.bdvpg.bdv.navigate.ViewerTransformAdjuster;
+import sc.fiji.bdvpg.sourceandconverter.display.BrightnessAutoAdjuster;
+import sc.fiji.bdvpg.services.BdvService;
+import sc.fiji.bdvpg.spimdata.importer.SpimDataFromXmlImporterAndRegisterer;
+
+/**
+ * Demonstrates visualisation of two spimData sources.
+ *
+ */
+public class SpimDataDisplayDemo
+{
+	public static void main( String[] args )
+	{
+		// Initializes static SourceService and Display Service
+		BdvService.InitScijavaServices();
+
+		// Gets active BdvHandle instance
+		BdvHandle bdvHandle = BdvService.getSourceAndConverterDisplayService().getActiveBdv();
+
+		// Import SpimData
+		new SpimDataFromXmlImporterAndRegisterer("src/test/resources/mri-stack.xml").run();
+		new SpimDataFromXmlImporterAndRegisterer("src/test/resources/mri-stack-shiftedX.xml").run();
+
+		// Show all SourceAndConverter associated with above SpimData
+		BdvService.getSourceAndConverterService().getSourceAndConverters().forEach( sac -> {
+			BdvService.getSourceAndConverterDisplayService().show(bdvHandle, sac);
+			new ViewerTransformAdjuster(bdvHandle, sac).run();
```

Comment:
Adjusting the view on each source is not necessary, but adjusting the view on all sources is not an available action on the moment. Maybe we should implement that ?

Reply:
Yes, we should! 
I made an issue: https://github.com/bigdataviewer/bigdataviewer-playground/issues/67